zproject/dev_settings.py

```diff
@@ -162,7 +162,7 @@
         }
     AUTHENTICATION_BACKENDS += ("zproject.backends.ZulipLDAPAuthBackend",)
 
-THUMBNAIL_IMAGES = True
+THUMBNAIL_IMAGES = False
```

Comment:
We should potentially just drop this line entirely, since it's the default?

I think we should probably also drop `THUMBNAIL_IMAGES = True` from `test_extra_settings.py`, though that may cause coverage failures which will require some further tweaking.

Reply:
Sure, I'll add it as a second commit.